tests/test_strtree.py

```diff
@@ -92,10 +92,11 @@ def test_pickle_persistence():
     """
     tree = STRtree([Point(i, i).buffer(0.1) for i in range(3)])
     pickled_strtree = pickle.dumps(tree)
-    print("pickled strtree:", repr(pickled_strtree))
-    unpickle_script_file_path = Path(__file__).parent / "unpickle-strtree.py"
-    subprocess.run(
+    unpickle_script_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "unpickle-strtree.py")
```

Comment:
Thanks for cleaning up this mess @tomplex . I'm not sure what happened in my merge of #915 but I clearly messed things up in a way that Travis-CI didn't notice.

Reply:
glad I could get it done!